app/upgrades/v1-beta.2/upgrades.go

```diff
@@ -0,0 +1,40 @@
+package v1_beta2
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	authkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+)
+
+func CreateUpgradeHandler(mm *module.Manager,
+	configurator module.Configurator,
+	authkeeper authkeeper.AccountKeeper,
+	bankkeeper bankkeeper.Keeper) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		ctx.Logger().Info(fmt.Sprintf("update start:%s", UpgradeName))
+		ctx.Logger().Info(fmt.Sprintf("update start test:%s", UpgradeName))
+		// add liquidity modules
+		// liquidity is auto init
```

Comment:
I think this is unnecessary.
plz remove it

Reply:
Fixed at https://github.com/UnUniFi/chain/pull/217/commits/d93ac0d3b29ef04c28737f4f0ca7ddd45fc474f1